examples/vault-ec2-auth/user-data-vault.sh

```diff
@@ -74,7 +90,10 @@ echo "$server_output" | head -n 3 | awk '{ print $4; }' | xargs -l /opt/vault/bi
 export VAULT_TOKEN=$(echo "$server_output" | head -n 7 | tail -n 1 | awk '{ print $4; }')
 
 # Enables AWS authentication
-/opt/vault/bin/vault auth enable aws
+# This is an http request, and sometimes fails, hence we retry
+retry \
+  "/opt/vault/bin/vault auth enable aws" \
+  "Trying to enable aws auth"
```

Comment:
Same as other comment. The other changes in this file are there just so this bit here is retried